### PR TITLE
Add new web page for the add-to-repo CLI command

### PR DESCRIPTION
**Pull request by groeges**
Signed-off-by: Steve Groeger <GROEGES@uk.ibm.com>

## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [ ] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Related Issues

#issue-number

### CODE REVIEW

File: content/docs/stacks/publish.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+This will create or update a repository index file using the --release-url argument value provided as the base URL required to reference the template archives. The index file will be determined from the the repository name, i.e. myrepository-index.yaml, and will be found in the .appsody/stacks/dev.local directory
+
+The generated repository index file will then need to be uploaded to the web hosting service.
+
+You can then provide the URL to this hosted repository index file to other Appsody uses who can add it to their Appsody repository list and then initialise a project using your stack. 
+
+## Publishing a stack using CI scripts
+
+To publish a stack using CI scripts, clone or copy the `appsody/stacks` Git repository. From the base directory
```
Comment (helenmasters) [inline content/docs/stacks/publish.md]: From the base directory?

Comment (groeges) [reply content/docs/stacks/publish.md]: Have removed this.

File: content/docs/stacks/publish.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+---
+title: Publishing Stacks
+---
+
+# Publishing Stacks
+
+Publishing a stack prepares the stack to be made available from a remote location so that it can be accessed by other individuals.
```
[the remark this answers is not in the document]
Comment (groeges) [reply content/docs/stacks/publish.md]: OK, can modify to use this.

**Comment by doveye** [comment]
I had rather a lot of review comments so I just re-wrote the topic ([publish.zip](https://github.com/appsody/website/files/3847673/publish.zip)) as that's probably less confusing :)  

Some questions:

1. It's not entirely clear to me what the "In order to publish a stack the stack needs to [...]" section is for - I interpreted it as an overview of the basic steps :)  However, does the "Contribute a stack to Appsody" process include generating Docker images and an index file?

2. In "Contribute a stack to Appsody", when the instructions say "The index file will be determined from the the repository name, i.e. myrepository-index.yaml, and will be found in the .appsody/stacks/dev.local directory", do you mean the index file **name** is determined from the repo name?

3. In "Publishing a stack using CI scripts":
- Is `<stack-id>` actually `<stack-name>`? "stack-id" makes me think it should be a number.
- It's not clear to me what gets created - the instructions say "a repository index files" but also "An index file is created for each repository that is listed in REPO_LIST", so is that one or multiple index files? I'm also not sure whether there's 1 stack container image and 1 template archive file for the single stack that you specify? Think this could be clarified.

**Pull request merged**